geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/NettyRedisServer.java

```diff
@@ -176,8 +176,8 @@ public void initChannel(SocketChannel socketChannel) {
         pipeline.addLast(new WriteTimeoutHandler(10));
         pipeline.addLast(ExecutionHandlerContext.class.getSimpleName(),
             new ExecutionHandlerContext(socketChannel, regionProvider, pubsub,
-                allowUnsupportedSupplier, shutdownInvoker, redisStats, backgroundExecutor,
-                redisPasswordBytes, getPort(), member));
+                allowUnsupportedSupplier, shutdownInvoker, redisStats, redisPasswordBytes,
```

Comment:
The `backgroundExecutor` removed here should also be removed from the constructor and can then also be removed from `GeodeRedisServer`.

Reply:
thanks!